### PR TITLE
feat: Added card component

### PR DESCRIPTION
**Pull request by xii-vi**
### Added Card Component

RA-One-Card

### CODE REVIEW

**Review by haarsh24 — APPROVED** [review]
Looks good. you can merge

**Comment by ark2002** [comment]
looks good . you can move ahead

File: pages/card.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                    style="width: 100%; height: 323px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (mohitdhatrak) [inline pages/card.html:55]: avoid inline style if possible, carbon issue hai then fine

File: pages/card.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                    style="width: 100%; height: 641px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <h4>Cards with shadow</h4>
+                <div class="card-wrapper shadow-card">
```
Comment (mohitdhatrak) [inline pages/card.html]: put 1 empty line between 2 card components html

File: stylesheet/card.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
+.card-wrapper {
+    display: flex;
+    max-width: 20rem;
+    background-color: var(--white);
```
Comment (mohitdhatrak) [inline stylesheet/card.css]: don't name css variable as a colour name else what's the use of variable, name like secondary-color or anything generic, if not then use hexcode maybe

File: stylesheet/card.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.text-card-content {
+    font-size: 0.8rem;
+    color: var(--black);
+    opacity: 0.6;
+}
+
+.horizontal-card-text-content {
+    background-color: var(--white);
```
Comment (mohitdhatrak) [inline stylesheet/card.css]: same with black white variables

**Review by mohitdhatrak — COMMENTED** [review]
Few suggestions otherwise looks good to me

**Review by devdeadviz — COMMENTED** [review]
Looks good to me!

File: pages/card.html (hunk 1 above)
Comment (xii-vi) [reply pages/card.html:55]: Yeah it is carbon issue

File: pages/card.html (hunk 2 above)
Comment (xii-vi) [reply pages/card.html]: Sure thanks for the input

File: stylesheet/card.css (hunk 3 above)
Comment (xii-vi) [reply stylesheet/card.css]: _Suggested Changes are made_

**Pull request merged**